README.md

```diff
@@ -1,8 +1,55 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql)
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Richard Polisuk . Jarvis Consulting
 
+Experienced Information Technology professional with system and infrastructure architecture experience currently pursuing software engineering opportunities. Professional experience with IBM Canada, CIBC, and TELUS. Academic experience from McGill University and Seneca College.
```

Comment:
Could you expand on this intro? Perhaps a brief overview of the sorts of work you pursued at those companies, the nature of your academic experience, and a short blurb about what you are looking to pursue in the future. Please DM Trevor if you'd like some samples.